### DISCUSSION TITLE
https://simeonat.github.io/blog/deno-graphql/

### DISCUSSION THREAD

**Discussion by giscus** [SimeonAT/simeonat.github.io#2] [General]
# https://simeonat.github.io/blog/deno-graphql/

In order to open my mind on what lies outside of the Node.js and npm ecosystem, I decided to give Deno a try.

https://simeonat.github.io/blog/deno-graphql/

**Comment by zeskorion** [comment]
a test

**Comment by sirapatp257** [comment]
And another test comment

**Comment by bdasgupta027** [comment]
Super comprehensive!

**Reply by gejpark** [reply]
Agreed.

**Comment by alidezhihui** [comment]
This is an awesome tutorial. I hope to see more in the future!

**Comment by Victoriakaey** [comment]
Good stuff!

**Comment by gejpark** [comment]
Interesting stuff!

**Comment by omgyukiel** [comment]
delectable content

**Comment by LeckyC** [comment]
The article is really helpful knowing the in and out of the subject. I leant something today!! Thanks.

**Comment by TylerMorton** [comment]
Auth go brrrrr.
